src/pages/TestK8s.tsx

```diff
@@ -69,6 +68,11 @@ const TestK8s: React.FC = () => {
         namespace,
       },
     };
+
+    const testConfigMetadata = {
```

Comment:
ConfigMaps don't exist anymore. We should bring over our Application changes if they want to keep this page.

Reply:
On second thought -- we can do this later if needed -- let us avoid this overhead for now.